internal/tokencache/cache_serialization_contract_test.go

```diff
@@ -38,155 +39,161 @@ func TestCacheSerializationContractUnmarshalJSON(t *testing.T) {
 	jsonFile, err := os.Open(testFile)
```

Comment:
These errors should be checked and errored/panicked on

Reply:
Fixed